Shared/Actions/ArchiveParams.cs

```diff
@@ -89,13 +107,14 @@ public ArchiveParams Prefixes(List<string> prefixes)
         public override void Check()
         {
             if ((m_publicIds == null || m_publicIds.Count == 0) &&
+                (m_fullyQualifiedPublicIds == null || m_fullyQualifiedPublicIds.Count == 0) &&
```

Comment:
@const-cloudinary , please add a validation for auto && fully qualified names. 
Also check the mutually exclusive behavior